package.json

```diff
@@ -186,7 +186,7 @@
 		"@babel/core": "^7.11.1",
 		"@babel/register": "^7.10.5",
 		"@babel/runtime": "^7.11.1",
-		"@signal-noise/stylelint-scales": "^1.3.0",
+		"@signal-noise/stylelint-scales": "1.4.0",
```

Comment:
Is there a reason why we have to lock in to this version?

Reply:
Good question! This is what showed up when I ran `yarn upgrade` for the package. I'm sure we could use any minor version after this, too.